tailwind.config.js

```diff
@@ -117,6 +117,9 @@ module.exports = {
       borderRadius: {
         '3xl': '1.25rem', // instead of 1.5rem, keeps the .25rem steps
       },
+      borderWidth: {
+        '6': '6px',
+      },
```

Comment:
In this case it makes sense, since it completes the ladder, but please check with me before adding stuff to the tailwind.config.